### DISCUSSION TITLE
Image not using all available height

### DISCUSSION THREAD

**Discussion by leandrobattochio** [QuestPDF/QuestPDF#784] [Q&A] [answered]
I am trying to make my image use all the height available. Just like the placeholder, but it is not taking all the available height. This is what is happening:

![image](https://github.com/QuestPDF/QuestPDF/assets/1385182/114a213d-4ce7-4e46-8897-e1d4ead6b0b5)

This is the code:

```
column
    .Item()
    .PaddingTop(10)
    .Border(1)
    .Table(table =>
    {
        table.ColumnsDefinition(column =>
        {
            column.RelativeColumn();
            column.RelativeColumn();
        });

        table.Header(header =>
        {
            header.Cell().BorderBottom(1).BorderRight(1).AlignCenter().Text("Antes").Bold();
            header.Cell().BorderBottom(1).AlignCenter().Text("Depois").Bold();
        });


        if (actionPlan.ImageBefore == null)
        {
            table
                .Cell()
                .Padding(10)
                .Height(450)
                .BorderRight(1)
                .Image(Placeholders.Image);
        }
        else
        {
            var bytes = Helpers.GetImageBytes(actionPlan.ImageBefore, _uploader);
            table
                .Cell()
                .Padding(10)
                .Height(450)
                .AlignCenter()
                .Image(bytes);
        }


        if (actionPlan.ImageAfter == null)
        {
            table
                .Cell()
                .Padding(10)
                .Height(450)
                .Image(Placeholders.Image);
        }
        else
        {
            var bytes = Helpers.GetImageBytes(actionPlan.ImageAfter, _uploader);
            table
                .Cell()
                .Padding(10)
                .Height(450)
                .AlignCenter()
                .Image(bytes);
        }

    });
```

Why is the placeholder image taking all available height while the dynamic image is not?

I already tried using FitArea() or even FitHeight(). None of them worked. Maybe I'm missing something else?

When DebugArea is enabled, I can see it takes all the available space that I want. But the image is not using all available space.

![image](https://github.com/QuestPDF/QuestPDF/assets/1385182/dbff62fe-2a19-48c2-b194-7d0080a1034d)

This is what happens when I try to .FitHeight()

![image](https://github.com/QuestPDF/QuestPDF/assets/1385182/afc751bf-2fef-4e53-a518-4631b2307dc6)

**Comment by MarcinZiabek** [comment, marked as answer]
`.Image("path/myFile.png").FitUnproportionally()` should work as expected :)

**Reply by leandrobattochio** [reply]
My image stops being rendered if I use FitUnproportionally. Weird. What is wrong?

![image](https://github.com/QuestPDF/QuestPDF/assets/1385182/955e3011-d425-418d-965f-f996916f5c35)

```
table
    .Cell()
    .Padding(10)
    .Height(250)
    .AlignCenter()
    .DebugArea()
    .Image(bytes).FitUnproportionally();
```

**Reply by leandrobattochio** [reply]
Well, I don't know why but drawing it just like this solved the issue:

```
                                table
                                    .Cell()
                                    .Padding(10)
                                    .Height(350)
                                    .Image(bytes).FitUnproportionally();
```

![image](https://github.com/QuestPDF/QuestPDF/assets/1385182/564f2936-85df-407b-b8d2-544e71de4052)


Thank you!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
